app/views/participants/fields/_family_history.html.erb

```diff
@@ -1,6 +1,9 @@
 <%= f.inputs :name => "Family History Information" do -%>
 
-    <%= f.input :first_generation, :as => :select, :label => "Did parents graduate from college?", :collection => { "No (student is first generation to college)" => true, "Yes (student is NOT first generation to college)" => false } %>
+    <%= f.input :first_generation, :as => :select, :collection => { "No (student is first generation to college)" => true, "Yes (student is NOT first generation to college)" => false } unless @current_user.person_type == "Participant" %>
+
+    <%= f.input :first_generation, :as => :select, :collection => { "No (I am first generation to college)" => true, "Yes (I am NOT first generation to college)" => false } unless @current_user.person_type != "Participant" %>
```

Comment:
This works in general, however the only difference between the two lines is the two words "student" and "I". So you could set a variable in the line before that's either "student" or "I" and use that variable in the collection string.